Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -480,7 +483,9 @@ private static GameObject CreateZoneMusic(PxVobZoneMusicData vob)
             var go = PrefabCache.TryGetObject(PrefabCache.PrefabType.VobMusic);
             go.SetParent(parentGosNonTeleport[vob.type], true, true);
             go.name = vob.vobName;
-            
+
+            go.layer = ConstantsManager.IgnoreRaycastLayer;
```

Comment:
Do we really need a separate Layer for this? Afaik Layers have other side effects like colliders won't work etc.

Reply:
This one might not need a collider, sure. But the question is more like: Do you know the drawbacks and is it really needed?

Reply:
this is the collider for music, we need to know when to change the track
it is intended to set the layer to this as we might get some weird interactions between the teleport raycast and collider

Reply:
Ok. Agreed. Can we at least add a comment to the ConstantsManager where we use this Layer, please. I have a feeling in 6 months we forget why we have an IgnoreRaycastLayer. Thanks.